src/components/Header/Header.jsx

```diff
@@ -31,7 +31,7 @@ const Header = (props) => {
       <div className={css.header__logo}>
         <img src={logo} width='172' height='48' alt='It`s done!'></img>
       </div>
-      <button onClick={onButtonShowClick} type='button' className={css.button_show}>{showCompletedBtn ? 'Hide completed' : 'Show completed'}</button>
+      {activeLink === 'canban' ? null : <button onClick={onButtonShowClick} type='button' className={css.button_show}>{showCompletedBtn ? 'Hide completed' : 'Show completed'}</button>}
```

Comment:
Длинная сложная строка. Можно так
```suggestion
      {
        activeLink === 'canban'
          ? null
          : <button onClick={onButtonShowClick} type='button' className={css.button_show}>{showCompletedBtn ? 'Hide completed' : 'Show completed'}</button>
      }
```